dockers/docker-config-engine-buster/Dockerfile.j2

```diff
@@ -1,5 +1,5 @@
 {% from "dockers/dockerfile-macros.j2" import install_debian_packages, install_python_wheels, copy_files %}
-FROM docker-base-buster
+FROM {{ docker_config_engine_buster_load_image }}
```

Comment:
i am not able to figure out what went wrong for this pr for armhf build, but can you split this pr into following pr so that the problem can be isolated.
1. use the load_image from the .mk file.
2. everything else.